README.md

```diff
@@ -43,24 +46,24 @@ challenge.filename # => ".well-known/acme-challenge/:some_token"
 # You can generate the body of the expected response.
 challenge.file_content # => 'string token and JWK thumbprint'
 
-# You can send no Content-Type at all but if you send one it has to be 'text/plain'.
+# You are not required to send a  Content-Type at all. If you do send one it has to be 'text/plain'.
```

Comment:
"... it has to be the one returned from this method." ? Else this comment kinda makes the entire method redundant.